packages/pigment-css-react/tests/testUtils.ts

```diff
@@ -52,7 +52,10 @@ export async function runTransformation(
       plugins: ['@babel/plugin-syntax-jsx'],
     },
     tagResolver(_source: string, tag: string) {
-      return require.resolve(`../exports/${tag}`);
+      if (_source.startsWith('@pigment-css/react')) {
+        return require.resolve(`../exports/${tag}`);
+      }
+      return null;
```

Comment:
@brijeshb42 without this change, all imports are considered pigment related APIs. `import * as React from 'react';` is not possible in the input file.

Reply:
All ears to better solution.